src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -134,6 +134,28 @@ private void authorizeResourceAdmin(final Resource resource) throws AccessDenied
         }
     }
 
+    private void authorizeResourceView(final Resource resource) throws AccessDeniedException {
+        try {
+            if (!adminService.isAdmin(AuthorizationService.Operation.ADMIN)) {
+                if (!authorizationService.isAuthorized(AuthorizationService.Operation.VIEW, resource)) {
```

Comment:
What will happen on plugun side when you call `authorizationService.isAuthorized(AuthorizationService.Operation.VIEW, resource)`? How will it decide that client has VIEW permission for a resource?

Reply:
ok, this is answered already